src/Integration.Vsix/CFamily/RulesLoader.cs

```diff
@@ -121,6 +121,8 @@ public class RuleMetadata
             [JsonProperty("defaultSeverity")]
             public Sonarlint.Issue.Types.Severity DefaultSeverity { get; set; }
 
+            [JsonProperty("compatibleLanguages")]
+            public string[] CompatibleLanguages { get; set; }
```

Comment:
Need to read more information from the metadata json file to know which language(s) the rule applies to.